lopper/assists/bmcmake_metadata_xlnx.py

```diff
@@ -324,14 +324,41 @@ def generate_hwtocmake_medata(sdt, node_list, src_path, repo_path_data, options,
             if sdt.tree['/'].propval('board') != ['']:
                 val = sdt.tree['/'].propval('board', list)[0]
                 fd.write(f'set(BOARD "{val}" CACHE STRING "BOARD")\n')
+
+            # Helpful in distinguishing minor differences within the same family device
+            # e.g. spartanuplus-spartanuplusaes, Versal_2ve_2vm (full and small variants)
+            variant_list = sdt.tree['/'].propval('variant', list)
```

Comment:
This variant property always present? we need to check for existence of this property, right?